Formula/w/whisperkit-cli.rb

```diff
@@ -1,7 +1,7 @@
 class WhisperkitCli < Formula
   desc "Swift native on-device speech recognition with Whisper for Apple Silicon"
   homepage "https://github.com/argmaxinc/WhisperKit"
-  url "https://github.com/argmaxinc/WhisperKit.git",
+  url "https://github.com/argmaxinc/WhisperKit/archive/refs/tags/v0.6.0.tar.gz",
       tag:      "v0.5.0",
       revision: "61df12a322aa9ff840a46f78115ad0ad93873e2e"
```

Comment:
Do these need removal now that we're using this github action that uses release archives instead of tags?

```suggestion
```

Reply:
Yeah they do.